examples/templates/aws-linux/README.md

```diff
@@ -10,6 +10,13 @@ tags: [cloud, aws]
 
 Pick this template in `coder templates init` and follow instructions.
 
+## Authentication
+
+This template assumes that coderd is run in an environment that is authenticated
+with AWS. For example, run `aws configure import` to import credentials on the
+system and user running coderd.  For other ways to authenticate [consult the
+Terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration).
```

Comment:
Currently, this isn't really be visible when creating a template via `coder templates init`. I added a mock "variable" in the Docker example, but I'm not really happy with this.

Any ideas? Perhaps it's also fair to rely on the default Terraform errors (+ these docs) to warn users.

Reply:
Yeah, I don't like the idea of hijacking parameters to guide end users.  If we can't expect users to read the README in the examples, and want do do some handholding, we should build out a guided `template init`